.github/workflows/build-packages.yml

```diff
@@ -0,0 +1,480 @@
+name: build-packages
+on: push
+env:
+  ANACONDA_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
+  ANACONDA_USER: ${{ secrets.ANACONDA_USER }}
+  NUM_OF_JOBS: 39
+defaults:
+  run:
+    shell: bash
+jobs:
+  libisl-linux:
+    runs-on: "ubuntu-16.04"
+    env:
+      PACKAGE: "lib/isl"
+      OS_NAME: "linux"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+      - uses: antmicro/litex-conda-ci@move-to-ghactions
+  binutils-or1k-linux:
```

Comment:
I'd put an empty line before each job, if it's legal. This, of course, applies to all repos.

Reply:
can we have comments? Maybe, just an idea, add a number of the job so that it's easy to add a new one at the end of the list?

Well, it will, of course, make it more difficult to add something in the middle

Reply:
I applied these suggestions to all repos.